web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -57,8 +57,19 @@ angular.module('oncokbApp')
                 }
                 scope.content.preStringO = scope.content.stringO;
                 $rootScope.$watch('reviewMode', function(n, o) {
-                    scope.reviewMode = n;
+                    if(n !== o) {
+                        scope.reviewMode = n;
+                    }
                 });
+                scope.calculateDiff = function() {
```

Comment:
Very minor, since we don't use this function in the template, can we just make a regular function inside the controller? Just reading a reply from this `http://stackoverflow.com/questions/15018821/should-i-always-bind-functions-to-scope-objects` we should try to make scope as clean as possible?